File: frontend/src/services/FamilyService.ts
import { axiosInstance } from './../utils/axios';

class FamilyService {
  public static async getFamilyList() {
    const response = await axiosInstance.get(
      'api/v1/family-list',
    );

    return response.data;
  }
}

export default FamilyService;

File: backend/src/main/java/com/ssafy/aptCom/api/service/FamilyService.java
package com.ssafy.aptCom.api.service;

import com.ssafy.aptCom.api.dto.response.FamilyDto;

import java.util.List;

public interface FamilyService {

    List<FamilyDto> getFamilyList(int userId, int aptId, String dong, String ho);
}
